Sources/Knit/Resolver.swift

```diff
@@ -13,4 +13,50 @@ public protocol Resolver: AnyObject {
 
     func unsafeResolver(file: StaticString, function: StaticString, line: UInt) -> Swinject.Resolver
 
+    init(_swinjectContainer: Swinject.Container)
+
+    static func inherits(from resolverType: Resolver.Type) -> Bool
```

Comment:
Include some comment docs on what is expected for someone to implement the override of this?